intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_utils.cpp

```diff
@@ -1148,15 +1159,18 @@ struct model_context* model_init_from_file(const char* path_model, struct model_
         /* .sl_kv = */ static_cast<int>(hparams.n_ctx),
     };
     const bool support_jblas_kv = ctx->support_jblas_kv && jblas_reordered_attn_fp32_support(&attn_shape);
+    fprintf(stderr, "%s: support_jblas_kv = %d\n", __func__, support_jblas_kv);
+
```

Comment:
Debug?

Reply:
also for inference. It's used to identify the MHA condistions.